pages/home.page.ts

```diff
@@ -0,0 +1,98 @@
+require('selenium-webdriver/chrome');
+const {By, Key, until } = require('selenium-webdriver');
+
+
+export class HomePage{
+    private driver;
+    private homeIcon; 
+    private cartText; 
+    private cartLink; 
+    private searchBox;
+    private womenBtn; 
+    private dressesBtn;
+    private tshirtsBtn;
+    private contactUsBtn;
+    private signInBtn; 
+    private newsletterFld;
+    private facebookIcon;
+    private twitterIcon;
+    private youtubeIcon;
+    constructor(driver){
+        this.driver = driver;
+    }
+
+    //#region methods
+    public async getCartText() {
+       this.cartText = await this.driver.findElement(By.className('ajax_cart_no_product')).getText();
+       return this.cartText;
+    }
+
+    public async clickOnCart(){
+        this.cartLink = await this.driver.findElement(By.xpath('/html/body/div/div[1]/header/div[3]/div/div/div[3]/div/a'));
+        await this.cartLink.click();
+    }
+
+    public async useSearcBox(input: string){
```

Comment:
I suggest to use better naming for variables that are sent to methods, in this example 'input' variable; it will be better if 'input' is 'searchedItem';
I see this 'input' vars in some other files, check them also